src/test/java/io/opentracing/contrib/jfrtracer/JFRTracerTest.java

```diff
@@ -53,9 +53,22 @@ public void basicEvent() throws IOException {
 			// Generate span
 			tracer.buildSpan("test span").start().finish();
 
+            //to be removed
```

Comment:
Instead of catching interrupted exception everywhere, just throw it or add a helper method.

Reply:
done